### PR TITLE
[plugins/aws][fix] Use a real aws cli instead of translating to boto3 APIs

### PR DESCRIPTION
**Pull request by meln1k**
# Description

# To-Dos

- [ ] Add test coverage for new or updated functionality
- [ ] Lint and test with `tox`
- [ ] Document new or updated functionality (someengineering/resoto.com#XXXX)

# Issues Fixed

- Fixes #XXXX

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

File: plugins/aws/resoto_plugin_aws/__init__.py [hunk 1]
```diff
@@ -198,100 +197,29 @@ def regions(self, profile: Optional[str] = None, partition: str = "aws") -> List
...
-
-        def create_client() -> AwsClient:
-            role = p.role or cfg.role
-            region = p.region or (cfg.region[0] if cfg.region else None)
-            profile = p.profile or (cfg.profiles[0] if cfg.profiles else None)
-            # possibly expensive call: account id is looked up if not provided
-            account = p.account or (cfg.account[0] if cfg.account else current_account_id(profile))
-            return AwsClient(cfg, account, role=role, profile=profile, region=region)
-
-        client = get_client(current_config(), resource) if resource else create_client()
```
Comment (aquamatthias) [inline plugins/aws/resoto_plugin_aws/__init__.py:272]: Region is coming from the resource.

**Review by aquamatthias — CHANGES_REQUESTED** [review]
Resource-specific properties need to be fetched from the resource.

File: plugins/aws/resoto_plugin_aws/__init__.py [hunk 2]
```diff
@@ -198,100 +197,36 @@ def regions(self, profile: Optional[str] = None, partition: str = "aws") -> List
...
-        op_name = get_name(pascalcase(p.operation), service_model.operation_names)
-        op: OperationModel = service_model.operation_model(op_name)
-        output_shape = op.output_shape.type_name
-        func_args = coerce_args(remaining, op)
-
-        # call single and treat result as list
-        response = client.call_single(p.service, p.operation, None, **func_args)
-        if response is None:
+        if resource:
+            ac = resource.account(self.graph)
```
Comment (aquamatthias) [inline plugins/aws/resoto_plugin_aws/__init__.py:201]: Does `self.graph` makes sense? Irritating from my POV.

Comment (meln1k) [reply plugins/aws/resoto_plugin_aws/__init__.py]: good point, fixed

**Review by aquamatthias — DISMISSED** [review]
thanks

**Review by aquamatthias — APPROVED** [review]
Thanks!

**Pull request merged**